packages/core/src/types/mitosis-node.ts

```diff
@@ -28,7 +28,7 @@ export type BaseNode = {
    * ```
    */
   bindings: {
-    [key: string]: { code: string; arguments?: string[] } | undefined;
+    [key: string]: { code: string; arguments?: string[]; type?: string } | undefined;
```

Comment:
Let's be precise with `type`'s possible values:

```suggestion
    [key: string]: { code: string; arguments?: string[]; type?: 'spread' } | undefined;
```

Reply:
done